docs/userGuide/syntax/annotations.md

```diff
@@ -0,0 +1,182 @@
+
+The x and y coordinates of each Annotate Point are relative to the image and are written in percentage of total width or height.
+
+<include src="codeAndOutput.md" boilerplate >
+<variable name="highlightStyle">html</variable>
+<variable name="code">
+
+<annotate src="../../images/annotateSampleImage.png" width="500" alt="Sample Image">
+  <!-- Minimal Point -->
+  <a-point x="25%" y="25%" content="This point is 25% from the left and 25% from the top" />
+  <!-- Customize Point Size (default size is 40px) -->
+  <a-point x="50%" y="25%" content="This point is 50% from the left and 25% from the top"  size="60"/>
+  <!-- Customize Point Header (default is empty) -->
+  <a-point x="75%" y="25%" content="This point is 75% from the left and 25% from the top"  header="This has a header"/>
+  <!-- Customize Point Color (default color is green) -->
+  <a-point x="33%" y="50%" content="This point is 33% from the left and 50% from the top"  color="red"/>
+  <!-- Customize Point Opacity (default opacity is 0.3) -->
+  <a-point x="66%" y="50%" content="This point is 66% from the left and 50% from the top"  opacity="0.7"/>
+  <!-- Customize Point Label (default is empty) -->
+  <a-point x="25%" y="75%" content="This point is 25% from the left and 75% from the top" label="1"/>
+  <!-- Customize Text Color (default color is black) -->
+  <a-point x="50%" y="75%" content="This point is 50% from the left and 75% from the top"  textColor="white" color="black" label="2" opacity="1"/>
+  <!-- Customize Font Size (default font size is 14) -->
+  <a-point x="75%" y="75%" content="This point is 75% from the left and 75% from the top"  fontSize="30" label="3"/>
+</annotate>
+</variable>
+</include>
+
+**Using triggers for Annotate Point** <br>
+
+Similar to **[popovers](../components/popups.md#popovers)**, Annotate Points also support different types of triggers and positions
+for users with different needs.
+
+<include src="codeAndOutput.md" boilerplate >
+<variable name="highlightStyle">html</variable>
+<variable name="code">
+
+<annotate src="../../images/annotateSampleImage.png" width="500" alt="Sample Image">
+  <!-- Default Trigger (click)-->
+  <a-point x="33%" y="33%" content="Lorem ipsum dolor sit amet" />
+  <!-- Set Trigger to hover focus -->
+  <a-point x="66%" y="33%" content="Lorem ipsum dolor sit amet" trigger="hover focus"/>
+  <!-- Set Popover Placement (click)-->
+  <a-point x="25%" y="66%" content="Popover on the left" placement="left"/>
+  <a-point x="50%" y="66%" content="Popover on the bottom" placement="bottom"/>
+  <a-point x="75%" y="66%" content="Popover on the right" placement="right"/>
+</annotate>
+</variable>
+</include>
+
+<br>
+
+**Displaying content as legends in Annotate Point**
+
+`<a-point>` allows users to display its content inside a popover or as
+a legend below the diagram or both. However, this only works if a label is
+provided for the point.
```

Comment:
```suggestion
`<a-point>` allows users to display its content !!inside a popover!! or !!as a legend below the diagram!! or !!both!!. However, a label must be provided for the point in order to display the content in both areas.
```

I think the "this" part is a bit confusing as to what it is referring to. I tried to reword it as above, but feel free to adjust as well.